src/models/converters/transactions/mod.rs

```diff
@@ -2,6 +2,8 @@ extern crate chrono;
 
 pub mod details;
 pub mod summary;
+
+#[cfg(test)]
```

Comment:
I am not sure this is how it works. This should probably be wherever you have the methods annotated with `#[test]`

Reply:
this works ... I tested it quite thoroughly ;)